src/subdomains/generic/kyc/services/name-check.service.ts

```diff
@@ -48,6 +48,11 @@ export class NameCheckService implements OnModuleInit {
       return RiskStatus.NOT_SANCTIONED;
     }
 
+    if (sanctionData.found_records.every((s) => !s.date_of_birth || s.date_of_birth.length === 0)) {
+      await this.createNameCheckLog(bankData, JSON.stringify(sanctionData), RiskStatus.MATCH_WITHOUT_BIRTHDAY);
```

Comment:
Can be simplified to `(s) => !s.date_of_birth?.length` if `date_of_birth` is an array?